experiment/popularity_community/initial_filling.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import asyncio
+import csv
+import logging
+import os
+import time
+
+from pony.orm import db_session, count
+
+from ipv8.peer import Peer
+from ipv8.peerdiscovery.discovery import RandomWalk
+from tool.tiny_tribler_service import TinyTriblerService
+from tribler_core.modules.popularity.popularity_community import PopularityCommunity
+
+_logger = logging.getLogger(__name__)
+
+TARGET_PEERS_COUNT = 20  # Tribler uses this number for some reason
+
+
+class ObservablePopularityCommunity(PopularityCommunity):
```

Comment:
Why is this class even in this PR? I thought it was a part of another PR?

Reply:
Seems to be moved from experiment/popularity_community/initial_filling/initial_filling.py